lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb

```diff
@@ -158,5 +159,14 @@ def api_token
     def api_url
       @api_url ||= MiqRegion.my_region.remote_ws_url
     end
+
+    def miq_request_task_url
+      result = {}
+      if TASK_TYPES.include?(@workspace.root['vmdb_object_type'])
```

Comment:
@mkanoor As discussed, we can remove the check for TASK_TYPES include here and instead check task_obj to be kind_of?(MiqRequestTask)